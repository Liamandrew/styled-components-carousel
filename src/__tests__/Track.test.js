import React from 'react';
import { render } from '@testing-library/react';
import Track from '../Track';

describe('Track', () => {
    it('should match the snapshot', () => {
        const { container } = render(
            <Track
                width={300}
                center={false}
                centerPadding={15}
                previousActive={1}
                active={0}
                infiniteActive={0}
                slideWidth={70}
                slideOffset={30}
            />,
        );

        expect(container).toMatchSnapshot();
    });

    it('should match the snapshot for center track', () => {
        const { container } = render(
            <Track
                width={300}
                center
                centerPadding={15}
                previousActive={1}
                active={0}
                infiniteActive={0}
                slideWidth={70}
                slideOffset={30}
            />,
        );

        expect(container).toMatchSnapshot();
    });
});
