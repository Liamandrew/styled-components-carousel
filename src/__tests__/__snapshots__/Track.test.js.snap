// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Track should match the snapshot 1`] = `
<div>
  <div
    class="sc-AxjAm cvCxQp"
    width="300"
  />
</div>
`;

exports[`Track should match the snapshot for center track 1`] = `
<div>
  <div
    class="sc-AxjAm coYcqt"
    width="300"
  />
</div>
`;
